addons/bus/models/bus.py

```diff
@@ -43,6 +49,24 @@ def channel_with_db(dbname, channel):
     return channel
 
 
+def get_notify_payloads(channels):
+    """
+    Generates the json payloads for the imbus NOTIFY.
+    Splits recursively payloads that are too large.
```

Comment:
For consistency, insert an empty line in between the docstring and the params
```suggestion
    Generates the json payloads for the imbus NOTIFY.
    Splits recursively payloads that are too large.

```